### ISSUE TITLE
whoogle.env problem

### ISSUE THREAD

**Issue by hxvy0** [benbusby/whoogle-search#909]
Whoogle.env not working on every setting. I use replit.

- My whoogle.env
`HOOGLE_CONFIG_NEAR=Edirne

#WHOOGLE_CONFIG_COUNTRY=TR

WHOOGLE_CONFIG_LANGUAGE=lang_tr

WHOOGLE_CONFIG_SEARCH_LANGUAGE=lang_en

WHOOGLE_CONFIG_DISABLE=1

WHOOGLE_CONFIG_URL=https://www.indexia.gq

WHOOGLE_CONFIG_STYLE=":root :root {
    /* LIGHT THEME COLORS */
    --whoogle-logo: #4c4f69;
    --whoogle-page-bg: #eff1f5;
    --whoogle-element-bg: #bcc0cc;
    --whoogle-text: #4c4f69;
    --whoogle-contrast-text: #5c5f77;
    --whoogle-secondary-text: #6c6f85;
    --whoogle-result-bg: #ccd0da;
    --whoogle-result-title: #7287fd;
    --whoogle-result-url: #dc8a78;
    --whoogle-result-visited: #e64553;

    /* DARK THEME COLORS */
    --whoogle-dark-logo: #cdd6f4;
    --whoogle-dark-page-bg: #1e1e2e;
    --whoogle-dark-element-bg: #45475a;
    --whoogle-dark-text: #cdd6f4;
    --whoogle-dark-contrast-text: #bac2de;
    --whoogle-dark-secondary-text: #a6adc8;
    --whoogle-dark-result-bg: #313244;
    --whoogle-dark-result-title: #b4befe;
    --whoogle-dark-result-url: #f5e0dc;
    --whoogle-dark-result-visited: #eba0ac;
}

#whoogle-w {
    fill: #89b4fa;
}

#whoogle-h {
    fill: #f38ba8;
}

#whoogle-o-1 {
    fill: #f9e2af;
}

#whoogle-o-2 {
    fill: #89b4fa;
}

#whoogle-g {
    fill: #a6e3a1;
}

#whoogle-l {
    fill: #f38ba8;
}

#whoogle-e {
    fill: #f9e2af;
}"
`

**Comment by KwisatzJim** [comment]
I'm also having a problem with the whoogle.env on replit.  I can't find any info on where the file should be on any install method.  I've tried multiple locations on pip and manual installs and none works.  Tried duplicating the template and editing it on replit but changes don't appear.  Not sure what I'm doing wrong.

**Comment by benbusby** [comment]
This should be fixed now. Previously the `WHOOGLE_DOTENV` var was required for whoogle.env to load, which was not intuitive. It now should load the env file if it exists in the top level app directory. I'm guessing this is what replit needs, although I don't use replit so let me know if there are any issues.

**Comment by KwisatzJim** [comment]
yes, I can confirm that it works properly on replit now.  thank you so much benbusby!

**Comment by KwisatzJim** [comment]
well the settings don't seem to persist between sessions and when doing the right click and 'search with whoogle' option, it doesn't load with the settings
